### ISSUE TITLE
A curiosity question

### ISSUE THREAD

**Issue by SimonMage** [jornadascobblemon/CobblemonMegas#6]
Just out of curiosity, have you started a development for this discontinued mod? If yes you will help a lot of servers

**Comment by 1mamute** [comment]
Yes, for the time being, the plan is to upgrade the **Fabric version** to Cobblemon 1.6.0 and fix some bugs.

This is a targeted effort to help the migration of our server to Cobblemon 1.6.0. We will discuss later if we will continue development or use [Flourish](https://modrinth.com/mod/flourish-cobblemon/versions). If we decide to drop the maintenance after the migration, I will archive the repo and notify accordingly.

Issues will be review in a best-effort basis, contributions are welcome.
